pyproject.toml

```
@@ -221,6 +221,11 @@ ignore = [
 
 [tool.pytest.ini_options]
 minversion = "6"
```

```suggestion
minversion = "8"
```

Let us bump this to `8` here because `6` is too old now (please update in the `[dev]` extras and the installation guide table in the docs as well).